util/dvsim/SimCfg.py

```diff
@@ -48,7 +48,7 @@ def __init__(self, flow_cfg_file, proj_root, args):
         self.max_waves = args.max_waves
         self.cov = args.cov
         self.cov_merge_previous = args.cov_merge_previous
-        self.profile = args.profile
+        self.profile = args.profile or '(cfg uses profile without --profile)'
```

Comment:
Can this be `None` instead? 
```suggestion
        self.profile = args.profile or None
```